### PR TITLE
Element mod.h

### PR DESCRIPTION
**Pull request by JensWehner**
ECP charges are now always loaded from ecpbasis and not from elements.h
qmatoms moved from ctp to xtp, breaks backwards compability
qmatoms converted from A to bohr, cxtp now only has nm,bohr, eV and Hartree

### CODE REVIEW

File: include/votca/xtp/aobasis.h [hunk 1]
```diff
@@ -104,7 +108,7 @@ class AOBasis
     
     int getMaxFunctions ( );
     
-private:
+  
     unsigned int _AOBasisSize;
```
Comment (baumeier) [inline include/votca/xtp/aobasis.h]: unsigned or unsigned int?

Comment (JensWehner) [reply include/votca/xtp/aobasis.h]: is the same thing, ```unsigned``` is short for ```unsigned int```

Comment (baumeier) [reply include/votca/xtp/aobasis.h]: I know, but some lines above you replace unsigned int by unsigned, so I wondered about consistency.

Comment (JensWehner) [reply include/votca/xtp/aobasis.h]: yeah in the end I want to completely move to ```unsigned```

File: src/libxtp/numerical_integration/radial_euler_maclaurin_rule.cc [hunk 2]
```diff
@@ -171,21 +171,21 @@ void EulerMaclaurinGrid::getRadialCutoffs(std::vector<ctp::QMAtom* > _atoms, Bas
...
                 int bidx = 0;
                 for (bit = _atoms.begin(); bit < _atoms.end(); ++bit) {
 
                     int _b_start = idxstart[bidx];
                     int _b_stop = idxstop[bidx];
-                    vec pos_b = (*bit)->getPos() * tools::conv::bohr2ang;
+                    const vec& pos_b = (*bit)->getPos();
                     // now do some overlap gymnastics
-                    double s_max = 10.0;
+                    double s_max =  std::numeric_limits<double>::max();
```
Comment (JoshuaSBrown) [inline src/libxtp/numerical_integration/radial_euler_maclaurin_rule.cc]: Nice!

**Comment by JensWehner** [comment]
We do not merge it into master but into the development branch

**Pull request closed without merging**